vrk-kpa/suomidigi
# SUOPA-659 Minievent

To test:
- `make fresh`
- Log in as a user which has no roles assigned
   - Go to your own page. Click on "Create new content" and select "New event"
   - You should end up on an ugly (non-themed) node add form
   - Fill in all the fields and click save
- On a different browser, log in as admin
   - Go to https://suomidigi.docker.sh/admin/content/approval-queue and check if there's your new event present.
   - Approve and publish. 
   - Go to the event
       - Check that the event has an URL ( /tapahtumat/2019/11/node-title)
       - Check that the layout looks like this one: https://xd.adobe.com/spec/ba71d258-eba5-4a06-57b8-e3b89749d908-cc40/screen/c444ca70-9595-413c-b484-84b86d597bc2/Suomidigi-tapahtuma-mini
